addpermissions.py

```diff
@@ -15,6 +15,9 @@
 
 This script requires the `gcloud` command line tool and the python
 `PyYaml` library.
+
+Example usage:
+  python3 addpermissions.py --users "andrea.frittoli@gmail.com, andrew.bayer@gmail.com, dlorenc@google.com, klewandowski@google.com, vdemeest@redhat.com, dibyajoti@google.com, sbws@google.com"
```

Comment:
typo: dibyajyoti :smile:

Reply:
@dibyom I'll let you do a follow-up :wink:

Reply:
https://github.com/tektoncd/plumbing/pull/248 :D